docs/chains/chaining-chainlets.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Chaining Chainlets
+description: "How two *chainlets* can be combined in a *chain*"
+---
+<Warning>This is a beta feature and subject to breaking changes.</Warning>
+
+# Table of Contents
+
+* [Introduction](/chains/intro)
+* [Getting Started](/chains/getting-started)
+* [Chaining Chainlets](/chains/chaining-chainlets)
+* [Dependencies & Resources](/chains/deps-and-resources)
+* [Reference](/chains/full-reference)
+
+
+In the [previous section](chains/getting-started), we created a simple chain with a single chainlet.
+In this section, we'll add a second chainlet to the chain. The toy example builds on top of
+the previous Hello World example, and prints out "Hello World" with a random number of exclamation points at the end.
+
+The computational graph for this chain looks like this:
+
+<img src="/images/second-chainlet.png" />
+
+{/* <Note> */}
+{/* In practice, for a chain this simple, you wouldn't need to use two chainlets. However, this example is useful for demonstrating how to use multiple chainlets in a chain. */}
+{/* </Note> */}
+
+# The Code
+
+Here's the code for this second chain. Exactly like the first example, we put this all in a single Python file called `hello.py`.
+
+```python
+import truss_chains as chains
+
+common_image = chains.DockerImage(
+    pip_requirements=["git+https://github.com/basetenlabs/truss.git@9be68f563d735289128cd3c8853bdf62ec03cef3"]
+)
+
+class GenerateRandomNumber(chains.ChainletBase):
+    """This chainlet generates a random number from 1 to `high_value`."""
+
+    remote_config = chains.RemoteConfig(docker_image=common_image)
+
+    def run(self, high_value: int) -> int:
+        import random
```

Comment:
Unless we explain *here* how to work with disjoint dependencies in the different images, we should not push down the import of random.